### PR TITLE
netlify deployed

### PR DESCRIPTION
**Pull request by IkeSteoger**

### CODE REVIEW

File: src/components/main.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import axios from 'axios';
+import Card from 'react-bootstrap/Card';
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (K1ng-T0ast) [inline src/components/main.js:7]: Nice job figuring out this part!

File: src/components/main.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                                        <Card.Text>Longitude: {this.state.lon}</Card.Text>
+                                    </Col>
+                                </Row>
+                                    <Card.Text><img src={this.state.map} alt="" /></Card.Text>
+                                </Card.Body>
+                            </Card>
+                            : <p className="text-center">Choose a city! (And double check the spelling is correct!)</p>
+                        }
+                        {
+                            this.state.error
```
Comment (K1ng-T0ast) [inline src/components/main.js:93]: Try creating a Bootstrap alert here!

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,35 @@
...
+.button {
+  background-image: linear-gradient(-180deg, #287A70, #7A6D31);
+  border-radius: 6px;
+  box-shadow: rgba(0, 0, 0, 0.1) 0 2px 4px;
+  color: #FFFFFF;
 }
 
-.App-link {
-  color: #61dafb;
+input[type="text"], textarea {
```
Comment (K1ng-T0ast) [inline src/App.css:21]: Whoa what is this doing? Is this just a pointer to an element?

File: src/components/main.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import axios from 'axios';
+import Card from 'react-bootstrap/Card';
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col';
```
Comment (K1ng-T0ast) [inline src/components/main.js:5]: You can also format this in one line FYI, "import { Col, Card, Container, Row } from 'react-bootstrap';

File: src/App.css (hunk 3 above)
Comment (IkeSteoger) [reply src/App.css:21]: I believe so!

**Pull request merged**